ci/docker/serve.build.Dockerfile

```diff
@@ -3,7 +3,6 @@
 ARG DOCKER_IMAGE_BASE_BUILD=cr.ray.io/rayproject/oss-ci-base_build
 FROM $DOCKER_IMAGE_BASE_BUILD
 
-ARG PYTHON_VERSION
```

Comment:
inherit from the base so no need to re-declare here

Reply:
I do not think docker build inherit build args.

if we are dropping this, we should just remove line 24-29

Reply:
w00t nice catch